WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipment Tracking Section/Add Tracking/ManualTrackingViewController.swift

```diff
@@ -134,12 +139,17 @@ private extension ManualTrackingViewController {
     }
 
     @objc func dismissButtonTapped() {
-        dismiss()
+        if viewModel.canCommit == false || hasUnsavedChanges {
```

Comment:
I think we don't care whether the view model can commit or not when dismissing the view, we only care about whether there are unsaved changes or not.